src/app/views/query-runner/QueryInput.tsx

```diff
@@ -29,6 +31,7 @@ export class QueryInput extends Component<IQueryInputProps, any> {
             role='listbox'
             selectedKey={selectedVerb}
             options={httpMethods}
+            disabled={mode === Mode.TryIt}
```

Comment:
TryIt mode will eventually support methods other than GET so perhaps we want to add a new mode called GetOnly to support this state.